.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Copied from https://github.com/storybookjs/storybook/blob/next/.prettierrc

Reply:
Is it possible to make this formatting work with the prettier config that we have already here?

https://github.com/chromaui/chromatic/blob/next/.eslintrc.js#L31

The eslint config is already setup with these params.

Reply:
Oops, didn’t see that. I will update it later tonight.